src/mumble/Settings.cpp

```diff
@@ -969,6 +968,20 @@ void Settings::legacyLoad(const QString &path) {
 	LOAD(iChatMessageMargins, "ui/ChatMessageMargins");
 	LOAD(bDisablePublicList, "ui/disablepubliclist");
 
+	if (settings_ptr->contains("ui/connectedquitbehaviour")) {
+		// Load the new connected quit option
+		LOADFLAG(connectedQuitBehavior, "ui/connectedquitbehaviour");
```

Comment:
This is dead code, as legacy settings will **never** contain this settings key (the key was introduced _after_ the new settings format has been introduced, meaning that every client that could save this setting will do so in the new format)